railties/lib/rails/templates/rails/tools/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h2>
+  Tools
+</h2>
```

Comment:
Any reason not to go inline? I wonder if the header should be a bit more descriptive than just "Tools".
```suggestion
<h2>Rails development tools</h2>
```

```suggestion
<h2>Rails dev tools</h2>
```

Reply:
Didn't go inline as I wanted to stay consistent with [routes](https://github.com/rails/rails/blob/2ccd3bff724e7c973a27ba4b4c8528730ad596f9/railties/lib/rails/templates/rails/info/routes.html.erb#L2) and [notes](https://github.com/rails/rails/blob/2ccd3bff724e7c973a27ba4b4c8528730ad596f9/railties/lib/rails/templates/rails/info/notes.html.erb#L39), which also didn't go inline. For the same reason I went with "Tools" instead of anything more descriptive.